driver-core/src/main/com/mongodb/connection/InternalStreamConnection.java

```diff
@@ -238,19 +230,19 @@ public boolean isClosed() {
     @Override
     public <T> T sendAndReceive(final CommandMessage message, final Decoder<T> decoder, final SessionContext sessionContext) {
         ByteBufferBsonOutput bsonOutput = new ByteBufferBsonOutput(this);
-        LazyCommandDocument lazyCommandDocument = new LazyCommandDocument(message, bsonOutput);
```

Comment:
LazyCommandDocument was a bug factory during the implementation so I removed it.

Reply:
Ack.